services/endpoint/endpoint.service.js

```diff
@@ -103,6 +103,21 @@ const description = `<p>
           the query string.
         </td>
       </tr>
+      <tr>
+        <td><code>logoSize</code></td>
```

Comment:
We've added this to the endpoint service docs, but it won't actually work. The `render()` method has not been updated to do anything with it.